test/server/cards/01_SOR/units/BobaFettDisintegrator.spec.js

```diff
@@ -0,0 +1,115 @@
+describe('Boba Fett Disintegrator', function() {
+    integration(function() {
+        describe('Boba Fett Disintegrator\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['vambrace-grappleshot'],
+                        groundArena: ['boba-fett#disintegrator'],
+                    },
+                    player2: {
+                        leader: 'luke-skywalker#faithful-friend',
+                        hand: ['lom-pyke#dealer-in-truths', 'consular-security-force', 'entrenched'],
+                    }
+                });
+            });
+
+            it('while attacking an exhausted unit that didn\'t enter play this round, deal 3 damage to the defender.', function () {
```

Comment:
nit, grammar: this currently reads as `Boba Fett Disintegrator's ability while attacking an exhausted unit that didn't enter play this round, deal 3 damage to the defender.`

Ideally would be something like `Boba Fett Disintegrator's ability, while attacking an exhausted unit that didn't enter play this round, should deal 3 damage to the defender`

Reply:
changed grammar of test text.